application/src/main/java/org/togetherjava/tjbot/commands/moderation/attachment/BlacklistedAttachmentListener.java

```diff
@@ -83,17 +89,17 @@ private Message createBaseResponse(String originalMessageContent, String dmMessa
     private List<String> getBlacklistedAttachmentsFromMessage(Message originalMessage) {
         return originalMessage.getAttachments()
             .stream()
-            .filter(attachment -> blacklistedFileExtensions
-                .contains(Objects.requireNonNull(attachment.getFileExtension()).toLowerCase()))
+            .filter(attachment -> blacklistedFileExtensions.contains(
+                    Objects.requireNonNull(attachment.getFileExtension()).toLowerCase(Locale.US)))
             .map(Message.Attachment::getFileName)
             .toList();
     }
 
     private boolean doesMessageContainBlacklistedContent(Message message) {
         List<Message.Attachment> attachments = message.getAttachments();
         return attachments.stream()
-            .anyMatch(attachment -> blacklistedFileExtensions
-                .contains(Objects.requireNonNull(attachment.getFileExtension()).toLowerCase()));
+            .anyMatch(attachment -> blacklistedFileExtensions.contains(
+                    Objects.requireNonNull(attachment.getFileExtension()).toLowerCase(Locale.US)));
```

Comment:
these `Objects.requireNonNull` are kinda obsolete. All they do is to trigger a NPE, which invoking a method on them would do anyways.

one could argue that it makes the intention a bit clearer but we had a discussion about that some time ago that we dont want to use it in such cases.